INSTALL

```diff
@@ -117,6 +111,12 @@
            OpenSSL version number with underscores instead of periods.
   Windows: C:\Program Files\OpenSSL or C:\Program Files (x86)\OpenSSL
 
+ The installation directory should be appropriately protected to ensure
+ untrusted users cannot make changes to OpenSSL binaries or files, or install
+ engines. If you already have a pre-installed version of OpenSSL as part of
```

Comment:
Just a cosmetical change. Feel free to adopt my suggestion or leave everything as it is.
```
 The installation directory should be write protected to ensure that
 unprivileged users cannot make changes to OpenSSL binaries or files, or install
```
(Personally, I find the term _untrusted_ too negative. Also, I replaced "_appropriately_ protected" by "_write_ protected`.)